src/algorithms/telemetry_decoder/gnuradio_blocks/beidou_b2a_telemetry_decoder_cc.h

```diff
@@ -0,0 +1,120 @@
+/*!
+ * \file beidou_b2a_telemetry_decoder_cc.h
+ * \brief Implementation of an adapter of a BEIDOU B2a CNAV2 data decoder block
+ * to a TelemetryDecoderInterface
+ * \note Code added as part of GSoC 2018 program
+ * \author Dong Kyeong Lee, 2018. dole7890(at)colorado.edu
+ *
+ * -------------------------------------------------------------------------
+ *
+ * Copyright (C) 2010-2015  (see AUTHORS file for a list of contributors)
+ *
+ * GNSS-SDR is a software defined Global Navigation
+ *          Satellite Systems receiver
+ *
+ * This file is part of GNSS-SDR.
+ *
+ * GNSS-SDR is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * GNSS-SDR is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with GNSS-SDR. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * -------------------------------------------------------------------------
+ */
+
+#ifndef GNSS_SDR_BEIDOU_L1_CA_TELEMETRY_DECODER_CC_H
```

Comment:
The same comment as before, this code seems copy/paste from GLONASS L1 CA implementation.